configuration/concepts/consultation_prenatale.csv

```diff
@@ -0,0 +1,51 @@
+dfb79477-ab61-47c4-b308-e1ab09757c18,,CIEL:162595,Vaginal delivery,Accouchement par voie naturelle,Vaginal delivery,Accouchement par voie naturelle,,,Finding,N/A,,,,,,,,,,,
+5066f452-5d8b-4e04-b815-0c212929e47e,,CIEL:1171,Cesarean,Césarienne,Cesarean,Césarienne,,,Misc,N/A,,,,,,,,,,,
+90950144-af82-460d-9393-e5e412445075,,,No workbirth (Abortion),Aucun accouchement (Avortement),No workbirth (Abortion),Aucun accouchement (Avortement),,,Misc,N/A,,,,,,,,,,,
+2a07d3b8-c3d5-4fae-ac89-e0a1dfdd9915,,CIEL:161033,Birth outcome during the last pregnancy,Issue de l'accouchement précédent,Birth outcome during the last pregnancy,Issue de l'accouchement précédent,,,Misc,Coded,,Vaginal delivery;Cesarean;No workbirth (Abortion),,,,,,,,,
+df9cc02b-b427-4262-b30a-7b973aef23fc,,CIEL:1536,Home,Domicile,Home,Domicile,,,Misc,N/A,,,,,,,,,,,
+023e0a9e-cf47-4a4d-9cf4-b2edfbe7fc33,,CIEL:1572,Place of delivery of the last pregnancy,Lieux de l'accouchement de la dernière grossesse,Place of delivery of the last pregnancy,Lieux de l'accouchement de la dernière grossesse,,,Misc,Coded,,St Jean Limbe Hospital;Good Samaritain Limbe Hospital;Justinien Cap Haitien Hospital;Sacred Heart Milot Hospital;HBCH Hospital;Gable Hospital;Borgne Hospital;Trou du Nord Hospital;Home;Other,,,,,,,,,
+05306cba-374b-40e8-ac40-4e8a510add61,,,Place of delivery of the last pregnancy data,Lieux d'accouchement des dernières données de grossesse,Place of delivery of the last pregnancy data,Lieux d'accouchement des dernières données de grossesse,,,ConvSet,N/A,,,Place of delivery of the last pregnancy; Other Health Centre,,,,,,,,
+50a37eff-81a3-4cef-a19c-1f85d4e690d6,,CIEL:1596,Date of most recent pregnancy delivery,Date du dernier accouchement,Date of most recent pregnancy delivery,Date du dernier accouchement,,,Misc,Date,,,,,,,,,,,
+e3892a91-1428-4816-8e83-834dcd738723,,,Any Previous Delivery Complications,,Any Previous Delivery Complications,Toute complication de livraison précédente,,,Misc,Coded,,Yes; No,,,,,,,,,
+402bba9c-ee24-45ac-947b-c574157e397d,,,Previous Delivery Complications,Complications d'accouchement précédentes,Previous Delivery Complications,Complications d'accouchement précédentes,,,Misc,Text,,,,,,,,,,,
+3b7d4768-8fc6-4a2d-9a83-05bd288b7108,,CIEL:1825,Number of living children,Nombre d'enfants vivants,Number of living children,Nombre d'enfants vivants,,,Misc,Numeric,,,,,,,,,,,
+6ed50023-d807-492f-a203-8ce87a942a52,,CIEL:1427,Date of last period,Date des dernières règles,Date of last period,Date des dernières règles,,,Misc,Date,,,,,,,,,,,
+6422a83f-acc6-4953-8239-f18a0b6d90a4,,CIEL:5596,Estimated date of delivery,Date prévue d'accouchement,Estimated date of delivery,Date prévue d'accouchement,,,Misc,Date,,,,,,,,,,,
+9e071d37-ee01-4785-a756-7e4059ffe736,,CIEL:1789,Gestational Age,Âge gestationnel,Gestational Age,L'âge gestationnel,,,ConvSet,N/A,,,Number of Weeks; Number of Days,,,,,,,,
```

Comment:
Let's remove the accent on "Age".

And also the short name is: "L'âge gestationnel". Let's put it "Age gestationnel"

Reply:
Done